src/lifecyrcle/Lifecycle.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
we don't use it since we've added React and Component as global variables in webpack